VMBackup/main/backuplogger.py

```diff
@@ -35,6 +35,10 @@ def __init__(self, hutil):
 
     def enforce_local_flag(self, enforced_local):
         self.enforced_local_flag_value = enforced_local
+        if (self.enforced_local_flag_value == False):
+            self.msg = self.msg + "================== Logs during Freeze ==============" + "\n"
+        elif (self.enforced_local_flag_value == True):
+            self.commit_to_local()
```

Comment:
add below log or something similar  here
            self.msg = self.msg + "================== End of Logs during Freeze ==============" + "\n"

Reply:
Good suggestion. Done.